index.bs

```diff
@@ -318,16 +332,17 @@ The <dfn method for=IdleDetector>start(|options|)</dfn> method steps are:
 1. [=In parallel=]:
```

Comment:
Could you instead make this whole block serial? I don't see anything that blocks. If the pass through the event loop is important, you should probably add a Note about why.

Reply:
This uses "in parallel"  because checking permission state is asynchronous in browser implementations. I'll add a note about that.